test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup_teardown/tasks/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+# tasks file for ec2
+
+- name: Create IAM role
+  iam:
+    iam_type: role
+    name: "{{iam_role_name}}"
+    state: present
+  register: role_output
+  tags: setup_infra
+
+- name: Assign a policy called SSM and S3 to the "{{iam_role_name}}"
+  iam_policy:
+    iam_type: role
+    iam_name: "{{iam_role_name}}"
+    policy_name: "{{iam_policy_name}}"
+    policy_document:  aws_ssm_integration_test_setup_teardown/files/custom_role.json
+    state: present
+  tags: setup_infra
+
+- pause:
+    seconds: 10
+  tags: setup_infra
+
+- name: Create ec2 instance
+  ec2:
+    instance_type: "{{instance_type}}"
+    image: "{{ami_id}}"
+    wait: "yes"
+    count: 1
+    region: us-east-2
+    instance_profile_name: "{{iam_role_name}}"
+    instance_tags:
+      name: krishna
+    state: present
+  register: output
+  tags: setup_infra
+
+- name: setting fact for bucket name using instance-id
+  set_fact:
+    bucket_name: "{{output.instance_ids[0]}}"
+    del_instance_id: "{{output.instance_ids[0]}}"
+    del_bucket_name: "{{output.instance_ids[0]}}"
+  tags: setup_infra
+
+- name: create var_to_delete.yml
```

Comment:
Please keep consistency for name of each and every module. Some where first letter is capital and some where it is small.